packages/trpc/server/routers/loggedInViewer/outOfOffice.handler.ts

```diff
@@ -10,12 +10,60 @@ import type { OOOEntryPayloadType } from "@calcom/features/webhooks/lib/sendPayl
 import sendPayload from "@calcom/features/webhooks/lib/sendPayload";
 import { getTranslation } from "@calcom/lib/server";
 import prisma from "@calcom/prisma";
+import { MembershipRole } from "@calcom/prisma/enums";
+import { WebhookTriggerEvents } from "@calcom/prisma/enums";
 import type { TrpcSessionUser } from "@calcom/trpc/server/trpc";
 
 import { TRPCError } from "@trpc/server";
 
-import type { TOutOfOfficeDelete, TOutOfOfficeInputSchema } from "./outOfOffice.schema";
-import { WebhookTriggerEvents } from ".prisma/client";
+import {
+  OutOfOfficeRecordType,
+  type TOutOfOfficeDelete,
+  type TOutOfOfficeEntriesListSchema,
+  type TOutOfOfficeInputSchema,
+} from "./outOfOffice.schema";
+
+// function getTeam() checks if there is a team where 'adminUserId' is admin or owner
+// and 'memberUserId' is a member.
+// If exists returns true.
+const getTeam = async (adminUserId: number, memberUserId: number) => {
```

Comment:
Please rename this function. getTeam() suggestes you are returning some team but it checks if current user has privilege to edit memberUser. You can then also remove the comment

Reply:
done